cmd/kubeadm/app/apis/kubeadm/v1beta2/defaults.go

```diff
@@ -40,7 +40,7 @@ const (
 	// DefaultCertificatesDir defines default certificate directory
 	DefaultCertificatesDir = "/etc/kubernetes/pki"
 	// DefaultImageRepository defines default image registry
-	DefaultImageRepository = "k8s.gcr.io"
+	DefaultImageRepository = "registry.k8s.io"
```

Comment:
@neolit123 If kubeadm is still `v1beta2` for 1.25, do we want to change the endpoint ?

Reply:
is there any image that could be pulled via `k8s.gcr.io` that could not be pulled via `registry.k8s.io`?